eng/Dependencies.props

```diff
@@ -75,6 +75,7 @@ and are generated based on the last package release.
     <LatestPackageReference Include="System.Runtime.InteropServices.RuntimeInformation" />
     <!-- System.Security.AccessControl should only be referenced in Dependencies.props and RepoTasks.csproj. -->
     <LatestPackageReference Include="System.Security.AccessControl" />
+    <LatestPackageReference Include="System.Security.Cryptography.Algorithms" />
```

Comment:
```suggestion
```

Reply:
Is this picked up automatically already?  AesGcm is in that package